app/src/apiClient.js

```diff
@@ -16,24 +14,32 @@ export const addTask = async (name) => {
   return await response.json();
 };
 
-//  export const testAuth = async (token) => {
-//    // FIXME: get JWT (need to resolve React-related error)
-//    const { getAccessTokenSilently } = useAuth0();
-//    const token = await getAccessTokenSilently();
-
-//    const response = await fetch("/api/test", {
-//      headers: {
-//        Authorization: `Bearer ${token}`,
-//      },
-//    });
-//    return await response.json();
-// };
+export const getUser = async (token, email) => {
+  const response = await fetch(`/api/user/${email}`, {
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  });
+  return await response.json();
+};
 
 export const getPlants = async (token, zone) => {
-  const response = await fetch(`/api/plants/${zone}`, {
+  const response = await fetch(`/api/plant/${zone}`, {
```

Comment:
just a small observation. You changed the api route from plants to plant I think to be consistent with the add plant route below. However this api/plant is in the getPlants function so maybe need to change function to getPlant singular to keep consistent?